inc/saimirror.h

```diff
@@ -346,6 +346,19 @@ typedef enum _sai_mirror_session_attr_t
      */
     SAI_MIRROR_SESSION_ATTR_UDP_DST_PORT,
 
+    /**
+     * @brief Attach a counter
+     *
+     * When it is empty, then packet hits won't be counted
```

Comment:
Can you reword to say something like: 

The number of packets that have been mirrored

Reply:
We need to specify the behavior of SAI_COUNTER_STAT_BYTES value. For example, there is ambiguity whether it is:

-   the mirrored packet payload bytes (after accounting truncation) or 
-   the incoming packet bytes

Reply:
Updated with your suggestion. Please let me know if this is sufficient or requires further revising.